test/e2e/capx_test.go

```diff
@@ -0,0 +1,20 @@
+//go:build e2e
+
+package e2e
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	"sigs.k8s.io/cluster-api/test/e2e"
+)
+
+var _ = Describe("CCM on a CAPX Cluster", Label("capx"), func() {
+	e2e.QuickStartSpec(ctx, func() e2e.QuickStartSpecInput {
```

Comment:
another option is to use following code and also verify if ccm is installed <pre>
By("Creating a workload cluster (no additional categories)", func() {
			testHelper.deployClusterAndWait(
				deployClusterParams{
					clusterName:           clusterName,
					namespace:             namespace,
					flavor:                flavor,
					clusterctlConfigPath:  clusterctlConfigPath,
					artifactFolder:        artifactFolder,
					bootstrapClusterProxy: bootstrapClusterProxy,
				}, clusterResources)
		})

		By("Checking cluster category condition is true", func() {
			testHelper.verifyConditionOnNutanixCluster(verifyConditionParams{
				clusterName:           clusterName,
				namespace:             namespace,
				bootstrapClusterProxy: bootstrapClusterProxy,
				expectedCondition: clusterv1.Condition{
					Type:   infrav1.ClusterCategoryCreatedCondition,
					Status: corev1.ConditionTrue,
				},
			})
		})
</pre>

Reply:
since quickstart wont check if ccm got installed or not

Reply:
https://github.com/nutanix-cloud-native/cluster-api-provider-nutanix/blob/main/test/e2e/basic_uuid_test.go

Reply:
we will also need one with clusterclass flavor as well then

Reply:
QuickStart will fail if CCM is not installed. So test passing actually ensures that CCM is installed and doing its job.